internal/cmd/workspaces.go

```diff
@@ -120,6 +131,201 @@ func lsWorkspacesCommand() *cobra.Command {
 	return cmd
 }
 
+func pingWorkspaceCommand() *cobra.Command {
+	var (
+		schemes []string
+		count   int
+	)
+
+	cmd := &cobra.Command{
+		Use:     "ping [workspace_name]",
```

Comment:
I think `[workspace_name]` should be `<workspace_name>` because the former usually means optional parameters and the latter means required parameters. For example, from the man page for git:

```
NAME
       git - the stupid content tracker

SYNOPSIS
       git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|-P|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>]
           <command> [<args>]
```